### PR TITLE
[DOCS] Update LTS overview

### PR DESCRIPTION
**Pull request by schavis**
Change terminology to "maintenance" rather than "support" to address customer confusion
Add timeline diagram
Remove redundant term "LTS support"
Create partial for image to ensure the same alt text is used any time the image is included

### CODE REVIEW

**Review by yhyakuna — APPROVED** [review]
Looks nice 👍

File: website/content/docs/enterprise/lts.mdx [hunk 1]
```diff
@@ -102,48 +101,36 @@ consider moving to a Vault Enterprise version with long-term support.
 
 ## What is long-term support?
 
-**Standard support** for Vault Enterprise includes:
+Long-term support offers extended maintenance (minor releases) for select, major
```
Comment (finnstech) [inline website/content/docs/enterprise/lts.mdx:104]: Could you move minor releases out of the parens into the sentence like "Long-term support offers extended maintenance via minor releases ..." something like that

File: website/content/docs/enterprise/lts.mdx [hunk 2]
```diff
@@ -102,48 +101,36 @@ consider moving to a Vault Enterprise version with long-term support.
...
-- high-risk security patches.
-
-After the first year, LTS Vault versions move from standard support to long-term
-support for an additional year.
+**standard maintenance** for a given Vault version lasts approximately 1 year.
+After the first year, LTS Vault versions move from standard maintenance to
+**extended maintenance** for an additional year with patches for bugs that may
+cause outages, critical vulnerabilities and exposures (CVEs), and high-risk
+security patches.
+
```
Comment (finnstech) [inline website/content/docs/enterprise/lts.mdx:117]: "with patches for bugs that may
cause outages, critical vulnerabilities and exposures (CVEs), and high-risk
security patches."
to 
"with patches for bugs that may cause outages and critical vulnerabilities and exposures (CVEs)."

**Review by finnstech — CHANGES_REQUESTED** [review]
left a few comments to tidy up grammar

File: website/content/docs/enterprise/lts.mdx (hunk 1 above)
Comment (schavis) [reply website/content/docs/enterprise/lts.mdx:104]: fixed

File: website/content/docs/enterprise/lts.mdx (hunk 2 above)
Comment (schavis) [reply website/content/docs/enterprise/lts.mdx:117]: sg. I wasn't sure if we wanted to differentiate between CVEs and "high-risk" security patches.

**Review by finnstech — APPROVED** [review]

**Pull request merged**